### PR TITLE
Fix proxy 2.6 jobs

### PR DESCRIPTION
**Pull request by jwendell**
By using the new image location

### CODE REVIEW

File: ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ tests:
       LOCAL_CPU_RESOURCES: "30"
       LOCAL_JOBS: "30"
       LOCAL_RAM_RESOURCES: "61440"
-      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:2.6
+      MAISTRA_BUILDER_IMAGE: quay-proxy.ci.openshift.org/openshift/ci:ci_maistra-builder_2.6
```
Comment (deepsm007) [inline ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml:33]: `MAISTRA_BUILDER_IMAGE: image_stream_tag`

File: ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml [hunk 2]
```diff
@@ -86,7 +86,7 @@ tests:
...
```
Comment (deepsm007) [inline ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml:89]: same as above

**Comment by jwendell** [comment]
closed in favor of https://github.com/openshift/release/pull/53111

**Pull request closed without merging**